builder/publish.go

```diff
@@ -132,3 +205,118 @@ func getDockerBuildxCommand(build dockerBuild) (string, []string) {
 func applyTag(index int, baseImage, tag string) string {
 	return fmt.Sprintf("%s:%s", baseImage[:index], tag)
 }
+
+func shrinkWrap(image, handler, lang string) error {
+	buildCmd := exec.Command(
```

Comment:
Remember that you are already in faas-cli - so the shrinkwrap code is available already. I'm quite surprised to see you trying to exec itself to perform this behaviour.

It seems like you copy/pasted the example for the function builder without really considering what this code was doing?